awesome/lists/lib/utils/awesome_list.py

```diff
@@ -30,6 +30,10 @@
     'Robotics': '🦿',
     'Self driving': '🚗',
     'Reinforcement learning': '🎮',
+    'System': '⚙️',
```

Comment:
Why are we collecting not AI-related courses? I don't think any of your newly added courses is correct.